package.json

```diff
@@ -35,7 +35,9 @@
     "test": "jasmine JASMINE_CONFIG_PATH=test/jasmine.json",
     "upload": "node build/upload.js"
   },
-  "dependencies": {},
+  "dependencies": {
+    "ietf-language-tag-regex": "0.0.5"
```

Comment:
You'll want to install this package as a dev dependency, rather than a regular dependency, which you can do with `npm i -D ietf-language-tag-regex`. [Here's](https://docs.npmjs.com/files/package.json#devdependencies) some information on the difference between regular and dev dependencies. In short, dev dependencies are packages that developers use to build the software, but end users don't need in order to run the software.